app/src/main/res/values/strings.xml

```diff
@@ -50,7 +50,7 @@
     <string name="menu_add_to_home_screen">Add to Home screen</string>
 
     <!-- Menu item: Shows information about features and changes that are new to this version -->
-    <string name="menu_whats_new">What\'s New</string>
+    <string name="menu_whats_new">What's New</string>
```

Comment:
Should this also be `’` instead of `'`, like https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/values/strings.xml#L134

Reply:
Yes.

Reply:
Good catch! (for once, I'm not the one chasing people around about this 😂 )